### PR TITLE
#33 -  Follow MLA guidelines for capitalization of titles

### PR DESCRIPTION
**Pull request by daceynolan**
Change all titles to follow MLA guidelines. 

closes https://github.com/triggermesh/docs/issues/33

### CODE REVIEW

**Comment by sebgoa** [comment]
perfect.

those seemingly small things and attention to details are highly appreciated

**Review by sebgoa — APPROVED** [review]

File: docs/guides/bridges/slack-to-confluent.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ You will need:
   - [Subscribe to Slack Events](#subscribe-to-slack-events)
   - [Testing the Integration](#testing-the-integration)
 
-## Create Confluent user
+## Create Confluent User
```
Comment (cab105) [inline docs/guides/bridges/slack-to-confluent.md:20]: Due to the way the ToC is generated, you will need to modify L14 to ensure the case matches as well.

File: docs/tm/registry.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-## Docker registry
+## Docker Registry
 
 Docker images are used to run functions code in Knative services. This means that image registry is important part of service deployment scheme. Depending on type of service, Knative controller may either only pull or also push service image from and to registry. Triggermesh CLI provides simple configuration interface to setup registry address and user access credentials.
 
-### Service from pre-build image
+### Service from Pre-Build Image
```
Comment (cab105) [inline docs/tm/registry.md:5]: Not directly related to the capitalization issue, but this should be `Pre-Built` instead of `Pre-Build`

**Review by cab105 — COMMENTED** [review]
Thank you for looking at this. Found one nit and an unrelated nit. Otherwise, looks good.

File: docs/tm/registry.md (hunk 2 above)
Comment (daceynolan) [reply docs/tm/registry.md]: Thank you :) I changed this

File: docs/guides/bridges/slack-to-confluent.md (hunk 1 above)
Comment (daceynolan) [reply docs/guides/bridges/slack-to-confluent.md:20]: I changed this :)

**Review by cab105 — APPROVED** [review]

**Comment by sameersbn** [comment]
rebased, merging...

**Pull request merged**